unit_tests/L2_testing/test_bundle.py

```diff
@@ -107,18 +196,391 @@ def test_verify_final_config_path(self):
+
+    def test_verify_final_ociversion(self):
+        #changes related to ociversion
+        final_ociVersion = exists(finalconfigdata, ['ociVersion'])
+        if not platform_generateCompliantConfig:
+            self.assertEqual(final_ociVersion,"1.0.2-dobby")
+        else:
+            self.assertEqual(final_ociVersion,"1.0.2")
+
+    def test_verify_final_hostname(self):
+        #changes related to hostname
+        platform_hostname = exists(platform_cfg, ['hostname'])
+        if platform_hostname:
+            final_config_hostname = exists(finalconfigdata,['hostname'])
+            if final_config_hostname is not None:
+                self.assertNotEqual(final_config_hostname.find(meta_app_id),-1)
+
+    def test_verify_final_annotations(self):
+        #changes related to annotations
+        final_config_annotations_stderr = exists(finalconfigdata,['annotations','run.oci.hooks.stderr'])
+        final_config_annotations_stdout = exists(finalconfigdata,['annotations','run.oci.hooks.stdout'])
+        if final_config_annotations_stderr is not None and final_config_annotations_stdout is not None:
+            self.assertEqual(final_config_annotations_stderr,"/dev/stderr")
+            self.assertEqual(final_config_annotations_stdout,"/dev/stdout")
+
+    def test_verify_final_hardware_maxram(self):
+        #changes related to hardware
+        platform_hardware_maxram = exists(platform_cfg, ['hardware', 'maxRam'])
+        if platform_hardware_maxram:
+            global meta_resources_ram
+            meta_resources_ram = exists(appmetadata, ['resources', 'ram'])
+            final_config_memory_limit =  exists(finalconfigdata,['linux','resources','memory','limit'])
+
+            print(meta_resources_ram)
+            meta_resources_ram_c = humanfriendly.parse_size(meta_resources_ram, binary=True)
+            platform_hardware_maxram_c = humanfriendly.parse_size(platform_hardware_maxram, binary=True)
+
+            if meta_resources_ram_c > platform_hardware_maxram_c:
+                self.assertEqual(platform_hardware_maxram_c,final_config_memory_limit)
+            else:
+                self.assertEqual(final_config_memory_limit,meta_resources_ram_c)
+
+    def test_verify_final_logging_pugin(self):
+        #changes related to logging
+        if exists(platform_cfg, ['logging','mode']) == 'file':
+            platform_logging_logdir = exists(platform_cfg, ['logging','logDir'])
+            final_config_logging_logdir = exists(finalconfigdata, ['rdkPlugins','logging','data','fileOptions','path'])
+            self.assertNotEqual(final_config_logging_logdir.find(platform_logging_logdir),-1)
+
+        if exists(platform_cfg, ['logging','limit']):
+            platform_logging_limit = exists(platform_cfg, ['logging','limit'])
+            final_config_logging_limit = exists(finalconfigdata,['rdkPlugins','logging','data','fileOptions','limit'])
+            self.assertEqual(final_config_logging_limit,paltform_logging_limit)
```

Comment:
typo: paltform_logging_limit should be platform_logging_limit

Reply:
Thanks, will update